ignite/chainconfig/config/config.go

```diff
@@ -69,8 +69,14 @@ type Client struct {
 	// TSClient configures code generation for Typescript Client.
 	Typescript Typescript `yaml:"typescript,omitempty"`
 
-	// Vuex configures code generation for Vuex stores.
-	Vuex Typescript `yaml:"vuex,omitempty"`
+	// Vuex configures code generation for Vuex stores. DEPRECATED and will be removed eventually.
+	Vuex Vuex `yaml:"vuex,omitempty"`
```

Comment:
We should follow Go recommendation for deprecation of functions, packages, fields and so on. [Here](https://github.com/golang/go/blob/689dc17793991746eb27ba4520d45af19a661f98/src/io/ioutil/ioutil.go#L25) is an example that follows what [Godoc](https://go.dev/blog/godoc) recommends for signaling deprecation.

```suggestion
	// Vuex configures code generation for Vuex stores.
	// 
	// Deprecated: Will be removed eventually.
	Vuex Vuex `yaml:"vuex,omitempty"`
```